### DISCUSSION TITLE
Fine-tuning original ControlNet Models

### DISCUSSION THREAD

**Discussion by basit-MT** [huggingface/diffusers#9226] [General]
Is it possible to Fine-tune original ControlNets instead of fine-tuning your own from scratch using SD as base models?

For example, can I perhaps, further Fine-tune "lllyasviel/control_v11f1p_sd15_depth"  for my own Custom data? Is this supported with Diffusers?

**Comment by tolgacangoz** [comment]
Is setting this argument what you are looking for:
https://github.com/huggingface/diffusers/blob/214990e5f239a70fd493c67ac193182a6eb7234f/examples/controlnet/train_controlnet.py#L261-L267

**Reply by basit-MT** [reply]
Thanks @tolgacangoz, completely overlooked it.
